dist/createEventEmitterProxy.d.ts

```diff
@@ -0,0 +1,20 @@
+/// <reference types="node" />
```

Comment:
Looks like you forgot to add something to `.gitignore`.

Reply:
Ah — I temporarily added `dist/` so I could test this with https://github.com/MetaMask/core/pull/1091, which this PR is blocking. I have confirmed this PR works with that one, so I'll revert the `dist/` commit.